app/src/main/java/com/jalasoft/routesapp/data/model/local/Location.kt

```diff
@@ -3,4 +3,11 @@ package com.jalasoft.routesapp.data.model.local
 data class Location(
     var latitude: Double,
     var longitude: Double
-)
+) {
+    fun toAndroidLocation(): android.location.Location {
+        val newLocation = android.location.Location("")
```

Comment:
Provide the __LocationManager.NETWORK_PROVIDER__ instead of an empty screen

Reply:
Now using LocationManager.NETWORK_PROVIDER.